src/Controller/LoginController.php

```diff
@@ -65,18 +63,18 @@ public function logoutAction() {
     /**
      * 
      * @param string $userName
-     * @param User $user
      * @return User OR null
      */
-    private function getUser($userName, $user) {
+    private function getUser($userName) {
         $sqlQuery = "SELECT password, salt, hashTimes FROM users WHERE userName='$userName'";
         $result = $this->connection->query($sqlQuery);
         if ($result->num_rows > 0) {
             $row = $result->fetch_assoc();
+            $user = new User();
             $user->setHashedPassword($row['password'])
                     ->setSalt($row['salt'])
                     ->setHashTimes($row['hashTimes']);
-            return TRUE;
+            return $user;
         } else {
             return FALSE;
```

Comment:
Man vistiek norisi šioje vietoje kad grąžintum null o ne false. Paskaityk https://www.php.net/manual/en/language.types.null.php ir https://stackoverflow.com/questions/137487/null-vs-false-vs-0-in-php. Jeigu visdar norėsi grąžinti false vietoj null - gausi argumentuoti kodėl